src/OrchardCore.Modules/OrchardCore.Security/Drivers/SecuritySettingsDisplayDriver.cs

```diff
@@ -45,15 +44,18 @@ public override async Task<IDisplayResult> EditAsync(SecuritySettings settings,
                 return null;
             }
 
+            settings = _httpContextAccessor.HttpContext.RequestServices.GetService<IOptions<SecuritySettings>>().Value;
```

Comment:
Why not inject this in the ctor?

Reply:
It could be injected, but I use the same way pattern of getting the user